### ISSUE TITLE
Integration of mail subscription

### ISSUE THREAD

**Issue by anurag03** [connectopensource/cto_website#5]
We need to integrate the mailing list subscription option with our website.
For beta version we need to add a link to our current mailing list page.

**Comment by ashwin-phadke** [comment]
Yes , what we can do is , I willl keep the localhost ready always, so if we start getting users atleast they have it visible , it can handle 50 connections at a time and once we are hosted then it's issue solved . We @connectopensource/infrastructure need a vte on this.
